examples/multiband_melgan/decode_mb_melgan.py

```diff
@@ -95,7 +95,7 @@ def main():
     config.update(vars(args))
 
     if config["format"] == "npy":
-        mel_query = "*-norm-feats.npy" if args.use_norm == 1 else "*-raw-feats.npy"
+        mel_query = "*-fs-after-feats.npy" if "fastspeech" in args.rootdir else "*-norm-feats.npy" if args.use_norm == 1 else "*-raw-feats.npy"
```

Comment:
revert to original code `mel_query = "*-norm-feats.npy" if args.use_norm == 1 else "*-raw-feats.npy"` since ur code is specific in ur case :D.

Reply:
`decode_fastspeech.py` and `decode_fastspeech2.py` are producing `{utt_id}-fs-before-feats.npy` and `{utt_id}-fs-after-feats.npy` files, as in

```py
for (utt_id, mel_before, mel_after, durations) in zip(
    utt_ids, masked_mel_befores, masked_mel_afters, duration_outputs
):
    # real len of mel predicted
    real_length = durations.numpy().sum()
    utt_id = utt_id.numpy().decode("utf-8")
    # save to folder.
    np.save(
        os.path.join(args.outdir, f"{utt_id}-fs-before-feats.npy"),
        mel_before[:real_length, :].astype(np.float32),
        allow_pickle=False,
    )
    np.save(
        os.path.join(args.outdir, f"{utt_id}-fs-after-feats.npy"),
        mel_after[:real_length, :].astype(np.float32),
        allow_pickle=False,
    )
```

So I modified `decode_mb_melgan.py` like that to run it with mels generated with fastspeech model.

Currently, `examples/fastspeech2/README.md` instructs to use following command to decode mels from folder ids.

```bash
CUDA_VISIBLE_DEVICES=0 python examples/fastspeech2/decode_fastspeech2.py \
  --rootdir ./dump/valid \
  --outdir ./predictions/fastspeech2.v1/ \
  --config ./examples/fastspeech2/conf/fastspeech2.v1.yaml \
  --checkpoint ./examples/fastspeech2/checkpoints/model-150000.h5 \
  --batch-size 8
```

which creates mel files in `./predictions/fastspeech2.v1/` directory, so checking `"fastspeech"` string in `args.rootdir` seemed simplest fix for me.

Reply:
How about adding an explicit flag like `args.mel_query` in `decode_mb_melgan.py` that defaults to `"*-raw-feats.npy"` or `"*-norm-feats.npy"` depending on `args.use_norm` just like now but gives ability to override them? It seems cleaner to me.

Reply:
ok i will add this after merge :D